workflows/joint_calling.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# To run the batch_workflow.py with the analysis runner. Assuming 
```

Comment:
Probably okay to just keep that in the `README.md`, otherwise it's hard to keep things in sync.

Reply:
Done!